tests/functional/api/basic_lti_launch_test.py

```diff
@@ -11,60 +11,28 @@
 
 
 class TestBasicLTILaunch:
-    SHARED_SECRET = "TEST_SECRET"
-    OAUTH_CONSUMER_KEY = "Hypothesis1b40eafba184a131307049e01e9c147d"
-    OAUTH_NONCE = "38d6db30e395417659d068164ca95169"
-    OAUTH_CLIENT = oauthlib.oauth1.Client(OAUTH_CONSUMER_KEY, SHARED_SECRET)
-
     @pytest.mark.usefixtures("http_intercept")
     def test_a_good_request_loads_fine(self, app, lti_params):
-        result = self.lti_launch(app, lti_params, status=200)
-
-        self.assert_response_is_html(result)
-
-    @classmethod
-    def assert_response_is_html(cls, response):
-        assert response.headers["Content-Type"] == Any.string.matching("^text/html")
-        assert response.html
-
-    @classmethod
-    def lti_launch(cls, app, params, status=200):
-        url = "/lti_launches"
-
-        params = cls.oauth_sign_params(url, params)
-
-        return app.post(
-            url,
-            params=params,
+        response = app.post(
+            "/lti_launches",
+            params=lti_params,
             headers={
                 "Accept": "text/html",
                 "Content-Type": "application/x-www-form-urlencoded",
             },
-            status=status,
-        )
-
-    @classmethod
-    def oauth_sign_params(cls, url, params):
-        params.update(
-            {
-                "oauth_consumer_key": cls.OAUTH_CONSUMER_KEY,
-                "oauth_nonce": cls.OAUTH_NONCE,
-                "oauth_timestamp": str(int(time.time())),
-            }
-        )
-        params["oauth_signature"] = cls.OAUTH_CLIENT.get_oauth_signature(
-            oauthlib.common.Request(f"http://localhost{url}", "POST", body=params)
+            status=200,
         )
 
-        return params
```

Comment:
I inlined `oauth_sign_params()` into the `lti_params()` fixture below. The `lti_params` fixture was only used once and the test passed its params through `oauth_sign_params()` after calling it